Microsoft.Toolkit.Uwp.UI.Controls/SlidableListItem/SlidableListItem.cs

```diff
@@ -153,6 +153,24 @@ public class SlidableListItem : ContentControl
         private DoubleAnimation _contentAnimation;
         private Storyboard _contentStoryboard;
 
+        private SwipeStatus _swipeStatus = SwipeStatus.NotRunning;
+
+        public SwipeStatus SwipeStatus
```

Comment:
What if let SwipeStatus be a DependencyProperty? So we can easily bind to it and got an implement of INotifyPropertyChange.